docs/api-inference/tasks/text-to-image.md

```diff
@@ -0,0 +1,103 @@
+## Text-to-image
+
+Generate an image based on a given text prompt.
+
+<Tip>
+
+For more details about the `text-to-image` task, check out its [dedicated page](https://huggingface.co/tasks/text-to-image)! You will find examples and related materials.
+
+</Tip>
+
+### Recommended models
+
+- [black-forest-labs/FLUX.1-dev](https://huggingface.co/black-forest-labs/FLUX.1-dev): one of the most powerful image generation models that can generate realistic outputs.
+- [latent-consistency/lcm-lora-sdxl](https://huggingface.co/latent-consistency/lcm-lora-sdxl): a powerful yet fast image generation model.
+- [Kwai-Kolors/Kolors](https://huggingface.co/Kwai-Kolors/Kolors): text-to-image model for photorealistic generation.
+- [stabilityai/stable-diffusion-3-medium-diffusers](https://huggingface.co/stabilityai/stable-diffusion-3-medium-diffusers): a powerful text-to-image model.
+
+This is only a subset of the supported models. Find the model that suits you best [here](https://huggingface.co/models?inference=warm&pipeline_tag=text-to-image&sort=trending).
+
+### API specification
+
+#### Inputs
+
+| Payload |   |    |
+| :--- | :--- | :--- |
+| **inputs** | _string, required_ | The input text data (sometimes called "prompt"). |
+| **parameters.guidance_scale** | _number, optional_ | For diffusion models. A higher guidance scale value encourages the model to generate images closely linked to the text prompt at the expense of lower image quality. |
+| **parameters.negative_prompt[]** | _string, optional_ | FOne or several prompt to guide what NOT to include in image generation. |
+| **parameters.num_inference_steps** | _integer, optional_ | For diffusion models. The number of denoising steps. More denoising steps usually lead to a higher quality image at the expense of slower inference. |
```

Comment:
I'm not a fan of the bunch of `parameters. ...`. Let's think if we can make something that keeps a clear difference while not being so repetitive

Reply:
for the record, I tried using nested lists

```
- **inputs** (_string, required_): The input text data (sometimes called "prompt").
- **parameters** (_object, optional_):
  - **guidance_scale** (_number, optional_): For diffusion models. A higher guidance scale value encourages the model to generate images closely linked to the text prompt at the expense of lower image quality.
  - **negative_prompt[]** (_string or string[], optional_): One or several prompts to guide what NOT to include in image generation.
  - **num_inference_steps** (_integer, optional_): For diffusion models. The number of denoising steps. More denoising steps usually lead to a higher quality image at the expense of slower inference.
  - **target_size** (_object, optional_):
    - **width** (_integer, optional_): The size in pixels of the output image.
    - **height** (_integer, optional_): The size in pixels of the output image.
  - **scheduler** (_string, optional_): For diffusion models. Override the scheduler with a compatible one.
```

resulting in 
![image](https://github.com/user-attachments/assets/17b2cc8c-589d-4950-a3ab-2c950113bef3)

Reply:
also tried nested table like this, which I'm really not a fan

```
| Payload |   |    |
| :--- | :--- | :--- | :--- | :--- |
| **inputs** |  |  | _string, required_ | The input text data (sometimes called "prompt"). |
| **parameters**  |  | |  |  |
|  | **guidance_scale**  || _number, optional_ | For diffusion models. A higher guidance scale value encourages the model to generate images closely linked to the text prompt at the expense of lower image quality. |
|  | **negative_prompt[]** | | _string, optional_ | One or several prompt to guide what NOT to include in image generation. |
|  | **num_inference_steps** | | _integer, optional_ | For diffusion models. The number of denoising steps. More denoising steps usually lead to a higher quality image at the expense of slower inference. |
|  | **target_size**  ||  |  |
|  |  | **width** | _integer, optional_ | The size in pixel of the output image. |
|  |  | **height** | _integer, optional_ | The size in pixel of the output image. |
|  | **scheduler**  | | _string, optional_ | For diffusion models. Override the scheduler with a compatible one. |
```

![image](https://github.com/user-attachments/assets/08d1e66c-1f92-4d0d-8088-b3fa4f345f18)

Reply:
Let's go with the last solution. It's ugly markdown-wise but the table looks ok. Can be changed in the future.